services/api/src/util.py

```diff
@@ -2,33 +2,51 @@
 import pytz
 import os
 
+
 # expects datetime string
-def format_date_utc(d):
-  if(not d):
-    return None
-  tmp = parse(d)
-  utc_tz = tmp.astimezone(pytz.UTC)
-  return (utc_tz.strftime("%Y-%m-%dT%H:%M:%S"))
+def format_date_utc(d, type="string"):
+    if not d:
+        return None
+    tmp = parse(d)
+    utc_tz = tmp.astimezone(pytz.UTC)
+
+    if type.upper() == "STRING":
+        return utc_tz.strftime("%Y-%m-%dT%H:%M:%S")
+    elif type.upper() == "DATETIME":
+        return utc_tz
+    else:
+        return None
+
 
 # expects datetime string
 def format_date_denver(d):
-  if(not d):
-    return None
-  tmp = parse(d)
-  denver_tz = tmp.astimezone(pytz.timezone('America/Denver'))
-  return (denver_tz.strftime('%m/%d/%Y %I:%M:%S %p'))
+    if not d:
+        return None
+    tmp = parse(d)
+    denver_tz = tmp.astimezone(pytz.timezone(os.getenv("TIMEZONE", "America/Denver")))
+    return denver_tz.strftime("%m/%d/%Y %I:%M:%S %p")
+
+
+# expects datetime object
+def format_date_denver_datetime(d):
```

Comment:
What's the difference between this new method and the `format_date_denver` above?

Reply:
Not being used anymore, I'll delete this.

Reply:
Addressed